Description: Backport "use system pyyaml modules" patch hunk from 1.10.0+dfsg-1
Origin: backport
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: netdata-1.9.0/python.d/python_modules/bases/loaders.py
===================================================================
--- netdata-1.9.0.orig/python.d/python_modules/bases/loaders.py
+++ netdata-1.9.0/python.d/python_modules/bases/loaders.py
@@ -8,11 +8,11 @@ from sys import version_info
 PY_VERSION = version_info[:2]
 
 if PY_VERSION > (3, 1):
-    from pyyaml3 import SafeLoader as YamlSafeLoader
+    from yaml import SafeLoader as YamlSafeLoader
     from importlib.machinery import SourceFileLoader
     DEFAULT_MAPPING_TAG = 'tag:yaml.org,2002:map'
 else:
-    from pyyaml2 import SafeLoader as YamlSafeLoader
+    from yaml import SafeLoader as YamlSafeLoader
     from imp import load_source as SourceFileLoader
     DEFAULT_MAPPING_TAG = u'tag:yaml.org,2002:map'
 
